src/app/data-fetchers/tequilapi-proposal-fetcher.js

```diff
@@ -64,11 +64,11 @@ class TequilapiProposalFetcher implements ProposalFetcher {
   }
 
   onFetchedProposals (subscriber: Callback<ProposalDTO[]>): void {
-    this._proposalSubscriber.subscribe(subscriber)
+    this._proposalPublisher.subscribe(subscriber)
   }
 
   onFetchingError (subscriber: Callback<Error>): void {
-    this._errorSubscriber.subscribe(subscriber)
+    this._errorPublisher.subscribe(subscriber)
```

Comment:
in some cases you use `Publisher.subscribe(subscriber)` in others `Publisher.subscribe(listener)`
in here I suggest 
```suggestion
    this._errorPublisher.subscribe(handleError)
```

Reply:
I want to keep this naming consistent - `handleError` is a bit confusing, because this subscriber can just log an error - I wouldn't call logging an error handling.